tests/test_smoke.py

```diff
@@ -1058,7 +1186,7 @@ def test_cancel_azure():
     run_one_test(test)
 
 
-# ---------- Testing `sky cancel` ----------
+@pytest.mark.no_lambda_labs  # TODO(ewzeng): fix nvidia-smi on Lambda Labs
```

Comment:
What is the problem with this test? Do we want to fix this before merging?

Reply:
Basically we need to download some packages on Lambda Labs to make `nvidia_smi` work on their vms. Let me see if I can fix now.

Reply:
Also this test is multi-node

Reply:
I kept the `no_lambda_labs` mark but updated the comment.